test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```

Comment:
Let's remove the `test` directory, untrack it and add it to the .gitignore. We are not going to use it, we will use `rspec`